decentraland/components/presentational/AccountCreationProgress.test.js

```diff
@@ -16,64 +16,68 @@ import { anAction } from "@helpers/testHelpers";
 describe("AccountCreationProgress", () => {
   it("initial state - before start account setup", async () => {
     const status = NOT_STARTED;
-    const action = null;
+    const actions = {};
 
     expect(
-      shallow(<AccountCreationProgress status={status} action={action} />)
+      shallow(<AccountCreationProgress status={status} actions={actions} />)
     ).toMatchSnapshot();
   });
 
   it("waiting for setup - waiting for account generation", async () => {
     const status = GENERATING_ACCOUNT;
-    const action = null;
+    const actions = {};
 
     expect(
-      shallow(<AccountCreationProgress status={status} action={action} />)
+      shallow(<AccountCreationProgress status={status} actions={actions} />)
     ).toMatchSnapshot();
   });
 
   it("waiting for setup - waiting for ETH funding", async () => {
     const status = FUNDING_WITH_ETH;
-    const action = anAction;
+    const actions = { FUNDING_WITH_ETH: anAction };
 
     expect(
-      shallow(<AccountCreationProgress status={status} action={action} />)
+      shallow(<AccountCreationProgress status={status} actions={actions} />)
     ).toMatchSnapshot();
   });
 
   it("waiting for setup - waiting for MANA funding and marketplace approval", async () => {
     const status = FUNDING_WITH_MANA_AND_APPROVING_MARKETPLACE;
-    const action = anAction;
+    const actions = {
+      FUNDING_WITH_MANA_AND_APPROVING_MARKETPLACE: null,
```

Comment:
Would this be the state of the actions storage in this case? I don't see where we're setting the action to `null` for the `FUNDING_WITH_MANA_AND_APPROVING_MARKETPLACE` account creation status. Maybe we should do that, even if the result would be the same whether or not it's set?

Reply:
You are right, the state `FUNDING_WITH_MANA_AND_APPROVING_MARKETPLACE` will never be in `actions` props. I'm almost sure that the test result will be the same, should I update the testcase removing it from `actions`?

Reply:
I agree that the result will be the same, but to avoid future confusion it seems worth making that small change while it's still fresh in our memory. Then I'll merge this!

Reply:
Done!

Reply:
Great, thanks! Resolving this.